RocketChat/Rocket.Chat
# fix: Render warning in Logs - (MessageType.render is deprecated. Use MessageType.message instead)

## Proposed changes (including videos or screenshots)

This PR aims to resolve the issue where a warning related to render function was logged which was - 
MessageType.render is deprecated. Use MessageType.message instead.

Before:
![image](https://github.com/RocketChat/Rocket.Chat/assets/124887084/122b5756-5565-4441-8472-fe21444c2078)

After:
![image](https://github.com/RocketChat/Rocket.Chat/assets/124887084/c8e3b29d-e196-4221-8a6e-fd06ed8e1ca4)


## Issue(s)
fixes [#28384](https://github.com/RocketChat/Rocket.Chat/issues/28384)

## Steps to test or reproduce
1. Start the application
2. See error in the logs

## Further comments
